docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -8,17 +9,560 @@
 .. _hpx_1_5_0:
 
 ===========================
-|hpx| V1.5.0 (TBD)
+|hpx| V1.5.0 (?)
 ===========================
 
 General changes
 ===============
 
+The main focus of this release is on APIs and C++20 conformance. We have added
+many new C++20 features and adapted multiple algorithms to be fully C++20
+conformant. As part of the modularization we have begun specifying the public
+API of HPX in terms of headers and functionality, and aligning it more closely
+to the C++ standard. All non-distributed modules are now in place, along with an
+experimental option to completely disable distributed features in HPX. We have
+also added an experimental asynchronous MPI executor. Lastly this release
+contains official |cmake| targets, performance improvements, and many bug fixes.
+
+* We have added the C++20 features ``hpx::jthread`` and ``hpx::stop_token``. We
+  have also added ``hpx::stable_sort`` based on Francisco Tapia's
+  implementation.
+* We have adapted existing sychronization primitives to be fully conformant
+  C++20: ``barrier``, ``latch``, ``counting_semaphore``.
+* We have started using customization point objects to make the corresponding
+  algorithms fully conformant to C++20 as well as to make algorithm extension
+  easier for the user. ``reduce``, ``copy``, ``all_of``/``any_of``/``none_of``,
+  ``fill``, ``count``, ``destroy``, ``move`` are using those CPOs.  We also have
+  adapted their corresponding ``hpx::ranges`` versions in this release,
+  deprecating the ``hpx::parallel`` ones.
+* In the perspective of continuing our work on making the hpx algorithms support
+  the ranges, we have adapted ``hpx::traits::is_range`` and others to use
+  ``hpx::traits::is_sentinel_for`` concept. We have adapted ``hpx::copy`` and
+  ``hpx::stable_sort`` algorithms.
```

Comment:
Are these the only algorithms that have been adapted? It may be easier to leave out the list here if it's just the same list as above.